x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -49,9 +50,17 @@ export function PageTitle({ alert }: PageTitleProps) {
 
   if (!alert) return <EuiLoadingSpinner />;
 
+  const showExperimentalBadge =
+    alert.fields[ALERT_RULE_CATEGORY] === 'Log threshold' ||
```

Comment:
Nit: @benakansara, Isn't it better to use the rule_type_id?

Reply:
It seems that the rule category and rule_type_id has 1:1 relation. Do you see any limitation with using rule category? I followed same logic as:

```{pageTitleContent(alert.fields[ALERT_RULE_CATEGORY])}```

Reply:
@benakansara, I think there is no issue with that. My concern is we are using `===` against strings with spaces 😅 
But if we use `ALERT_RULE_TYPE_ID `, we can do it without hard-coded strings. 
```suggestion
    alert.fields[ALERT_RULE_TYPE_ID] === LOG_DOCUMENT_COUNT_RULE_TYPE_ID
 ```

Reply:
@fkanout here is the PR to use `ALERT_RULE_TYPE_ID`. I created the const (`LOG_DOCUMENT_COUNT_RULE_TYPE_ID`, etc.) in the same file as these are defined within `infra` plugin.

https://github.com/elastic/kibana/pull/159979